### ISSUE TITLE
Wrong metadata for lens

### ISSUE THREAD

**Issue by nullhack** [darktable-org/darktable#12559]
**Introduction**

I just realized that photos taken with a RF70-200mm f/2.8 are wrongly assigned as RF70-200mm f/4
I don't know if anyone else had the same issue and I don't have the rented lens anymore to reproduce, but it may be some wrong entry in the database.
I am including a screenshot showing a photo taken at f/2.8 and the metadata showing the wrong name (f/4) for the lens.
Is there a way to change the metadata for the lens from the interface or a work around ?

**Screenshots**
![image](https://user-images.githubusercontent.com/11466701/192744418-66fbd216-f6c0-478d-b72e-70ea4a1baef0.png)

**Platform**

* darktable version : 3.8.1
* OS                : Ubuntu 22.04
* gcc               : gcc (Ubuntu 11.2.0-19ubuntu1) 11.2.0

Thank you all the contributors.

**Comment by kmilos** [comment]
I believe this was [fixed](https://github.com/darktable-org/darktable/pull/11584) already, please try the latest 4.0.1 version.

Also, please search before opening new issues.

**Comment by nullhack** [comment]
Thank you kmilos for the fast reply. Sorry I didn't make it more explicit, but I tried to search solutions before opening the issue, leaving this as last resource.
 
Searching on google and some foruns for I could not find anyone having the same issue, in github searching for RF related issues, I could not guess that 'Try to handle Canon RF lenses differently to legacy ones' was remotely related to manual metadata edits, so I decided to ask if anyone has had the same problem and if there's any work around for this.

Unfortunately, changing to 4+ is not an option at the moment as the package in apt still 3.8.1. I tried to install the one from snap (that is at 4+) but it lacks access to system resources because containerization and some lua scripts were broken because of that.

Thank you for the answer, I think the only option for me is to wait until apt gets updated in stable.
